lang/ilias_en.lang

```diff
@@ -17089,11 +17089,11 @@ trac#:#user_total#:#User Total
 trac#:#view_mode#:#View Mode
 tstv#:#tstv_create#:#Create Test Certificate
 tstv#:#tstv_create_info#:#Select a completed test to generate a certificate for it.
-ui#:#datatable_multiactionmodal_actionlabel#:#operation
+ui#:#datatable_multiactionmodal_actionlabel#:#Action for all entries
 ui#:#datatable_multiactionmodal_buttonlabel#:#Execute
-ui#:#datatable_multiactionmodal_listentry#:#apply to all objects
-ui#:#datatable_multiactionmodal_msg#:#<b>Caution!</b> multiple objects affected.
-ui#:#datatable_multiactionmodal_title#:#Operation on multiple objects
+ui#:#datatable_multiactionmodal_listentry#:#Actions for Complete Table
```

Comment:
I am not sure if 'complete' or 'entire' is the right choice. I'll ask Professor Potter ...

Reply:
Hi @matthiaskunkel 
are there any new insights?

Reply:
Here is the final feedback:
ui#:#datatable_multiactionmodal_buttonlabel#:#Apply
( we do not execute humans or things in ILIAS ;-) – and if possible, I would relabel the ID to 'datatable_multiactionmodal_apply' which is more semantic than 'buttonlabel' and we want to use semantic labels, right? )
ui#:#datatable_multiactionmodal_listentry#:#Actions for Entire Table

Reply:
Hello @lscharmer, can you take care of this please?

Reply:
`datatable_multiactionmodal_buttonlabel` is renamed to `datatable_multiactionmodal_apply`.
`Execute` is replaced with `Apply`.
`Actions for Complete Table` is renamed to `Actions for Entire Table`.